app/hub/Views/SideNavigationView/SideNavigationView.jsx

```diff
@@ -96,7 +99,13 @@ const SideNavigationView = (props) => {
 
 	return (
 		<div className={containerClassNames}>
-			<NavLink to="/" className={topClassNames} />
+			<a
+				href={GHOSTERY_BASE_URL}
+				aria-label="Ghostery website"
```

Comment:
Nice `aria-label` add!

Reply:
Thanks! It helps that the linter yells at you if you don't include one when you don't have text for your `a` tag, haha.